sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -69,6 +69,7 @@ def configure_azure_monitor(**kwargs) -> None:  # pylint: disable=C4758
      `{"azure_sdk": {"enabled": False}, "flask": {"enabled": False}, "django": {"enabled": True}}`
      will disable Azure Core Tracing and the Flask instrumentation but leave Django and the other default
      instrumentations enabled.
+    :keyword ~opentelemetry.sdk.resources.Resource] resource: OpenTelemetry Resource object.
```

Comment:
```suggestion
    :keyword ~opentelemetry.sdk.resources.Resource resource: OpenTelemetry Resource object.
```

Reply:
It might be worth clarifying here the overwrite behaviour as you described in the readme.